packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-component-builder/plugins/packages.ts

```diff
@@ -19,6 +19,11 @@ class PackagesPlugin implements ComponentBuilderPlugin {
      *      }
      *    ]
      *  }
+     * Or you can register all components from a path:
```

Comment:
Consider adding this comment to React template (to keep template parity): https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/react/scripts/generate-component-builder/plugins/packages.js